pkg/proxy/ipvs/proxier.go

```diff
@@ -274,6 +274,19 @@ func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
 // Proxier implements ProxyProvider
 var _ proxy.ProxyProvider = &Proxier{}
 
+// ParseExcludedCIDRs parses the input strings and returns net.IPNet
+// The validation has been done earlier so the error condition will never happen under normal conditions
+func ParseExcludedCIDRs(excludeCIDRStrs []string) []*net.IPNet {
```

Comment:
Can this be an unexported function? Doesn't seem like we're using it outside this package.

Reply:
This is used in test

Reply:
Tests in the same package can call unexported functions right?